app/build.gradle

```
@@ -38,4 +39,13 @@ android {
 
 dependencies {
     implementation project(path: ':base')
+
+    // Hilt
+    implementation "com.google.dagger:hilt-android:$hilt_version"
+    implementation 'androidx.appcompat:appcompat:1.3.1'
+    implementation 'com.google.android.material:material:1.4.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.0'
+    kapt "com.google.dagger:hilt-android-compiler:$hilt_version"
+    implementation 'androidx.hilt:hilt-lifecycle-viewmodel:1.0.0-alpha03'
+    kapt 'androidx.hilt:hilt-compiler:1.0.0'
```

적어준 주석대로 dependency 관리하면 좋을 것 같아!

```suggestion
    // Hilt
    implementation "com.google.dagger:hilt-android:$hilt_version"
    kapt "com.google.dagger:hilt-android-compiler:$hilt_version"
    implementation 'androidx.hilt:hilt-lifecycle-viewmodel:1.0.0-alpha03'
    kapt 'androidx.hilt:hilt-compiler:1.0.0'
```

그리고 아마 `constraintlayout`이나 `appCompat` 라이브러리 들은 base 모듈에서 api로 이미 제공해주고 있어서 제거해줄 수있어!

---

ㅋㅋㅋㅋㅋㅋ이거 재민이가 올려줬던 프로젝트 클론해서 그대로 복붙한거였는데 material이랑 constraintlayout들어가있는지 몰랐당 고마오!!